change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -1854,12 +1856,20 @@ PROCEDURE VALIDATECONFIG
      PARAMVALIDATIONERROR=TRUE
      PARAMVALIDATIONERRORCHR="No email detail lines specified for member email."
```

Comment:
I suggest doing validation on the length of what goes in fields:
Line 1532 PARAMEMAILSUBJECT=PARAMVALUE
and
Line 1595 PARAMEMAILTEXT(EMAILLINECOUNT)=PARAMVALUE

Let me know if you, you think the Banno Debug message in the Email Send error
is enough.